test/api-test.js

```diff
@@ -396,6 +396,17 @@ describe('IP library for node.js', function() {
       assert.equal(ip.toLong('127.0.0.1'), 2130706433);
       assert.equal(ip.toLong('255.255.255.255'), 4294967295);
     });
+    it('should be able handle bad input', function() {
+      assert.throws(function () {
+        ip.toLong('fd12:3456:789a:1::1');
+      }, Error);
+      assert.throws(function () {
+        ip.toLong(null);
+      }, Error);
+      assert.throws(function () {
+        ip.toLong(undefined);
```

Comment:
`false`, empty string `''`, `Function` ? :)

Reply:
I would end up testing here this `.isV4Format` more as in the end this will be used to validated input - so just added few possible values that can perhaps happens as bad input other than ipv6